packages/std/src/ibc/callbacks.rs

```diff
@@ -15,12 +15,34 @@ use crate::{Addr, IbcAcknowledgement, IbcPacket, Uint64};
 ///
 /// # Example
 ///
+/// Using [`TransferMsgBuilder`](crate::TransferMsgBuilder):
 /// ```rust
 /// use cosmwasm_std::{
-///     to_json_string, Coin, IbcCallbackRequest, IbcMsg, IbcSrcCallback, IbcTimeout, Response,
+///     to_json_string, Coin, IbcCallbackRequest, TransferMsgBuilder, IbcSrcCallback, IbcTimeout, Response,
 ///     Timestamp,
 /// };
+/// # use cosmwasm_std::testing::mock_env;
+/// # let env = mock_env();
+///
+/// let _transfer = TransferMsgBuilder::new(
```

Comment:
```suggestion
/// let _msg = TransferMsgBuilder::new(
```

here I think it's important to highlight we get the simple message out, not some obscure data structure